specification/ai/ModelClient/models/chat_completions.tsp

```diff
@@ -531,7 +531,7 @@ model FunctionDefinition {
 
   #suppress "@azure-tools/typespec-azure-core/no-unknown" "External API shape takes an arbitrary json"
   @doc("The parameters the function accepts, described as a JSON Schema object.")
-  parameters?: unknown;
+  parameters?: Record<unknown>;
```

Comment:
You'll need to add this line to ` tspconfig.yaml` to suppress `npx tsv .` error

```
linter:
  extends:
    - "@azure-tools/typespec-azure-rulesets/data-plane"
  disable:
    "@azure-tools/typespec-azure-core/bad-record-type": "We do want to use Record<unknown>, and not Record<string>"
```

Reply:
Looks like it's already there